topcoffea/cfg/data_samples_NDSkim.cfg

```diff
@@ -58,7 +58,7 @@ root://deepthought.crc.nd.edu/
 ../../topcoffea/json/data_samples/2017/DoubleMuon_F-UL2017_NDSkim.json
 
 ../../topcoffea/json/data_samples/2017/MuonEG_B-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/MuonEG_C-UL2017_NDSkim.json
+../../topcoffea/json/data_samples/2017/MuonEG_C-UL2017_NDSkim.json#
```

Comment:
Any reason for this trailing `#`? If not then maybe we can remove it.

Reply:
removed